examples/src/main/java/io/undertow/examples/ssl/HelloWorldSSL.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * JBoss, Home of Professional Open Source. Copyright 2014 Red Hat, Inc., and individual contributors as indicated by
+ * the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.undertow.examples.ssl;
+
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.nio.file.Paths;
+import java.security.KeyStore;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.net.ssl.KeyManager;
+import javax.net.ssl.KeyManagerFactory;
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.TrustManager;
+import javax.net.ssl.TrustManagerFactory;
+
+import org.xnio.Options;
+import org.xnio.Sequence;
+
+import com.google.common.io.Resources;
+
+import io.undertow.Undertow;
+import io.undertow.UndertowOptions;
+import io.undertow.examples.UndertowExample;
+import io.undertow.server.HttpHandler;
+import io.undertow.server.HttpServerExchange;
+import io.undertow.util.Headers;
+
+/**
+ * @author Stuart Douglas
+ */
+@UndertowExample("Hello World SSL")
+public class HelloWorldSSL {
+
+    public static void main(final String[] args) throws Exception {
+        final SSLContext sslContext = SSLContext.getInstance("TLSv1.2");
+        sslContext.init(getKeyManagers(System.getProperty("keyStorePath")),
+                        getTrustManagers(System.getProperty("trustStorePath")), null);
+
+        final List<String> cipherSuites = new ArrayList<>();
+        cipherSuites.add("TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256");
```

Comment:
Instead of adding a new example I think it may make more sense to just change the HTTP/2 example to use this configuration (as HTTP/2 won't work if a weak cipher is selected)